webpack.config.js

```diff
@@ -0,0 +1,81 @@
+const webpack = require('webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = env => {
+	env = env || {};
+	const config = {
+    mode: env.mode || 'production',
+		entry: './src/index.js',
+		output: {
+			path: path.resolve(__dirname, 'dist'),
+			filename: 'bundle.js'
+    },
+		module: {
+			rules: [
+				{
+          test: /\.html$/,
+					use: [
+						{ loader: 'babel-loader' },
+						{
+              options: {
+                processStyleLinks: true
+              },
+              loader: 'polymer-webpack-loader'
+            }
+					]
+        },
+				{
+          test: /polymer\.html$/,
```

Comment:
So I don't think this is actually doing anything as it doesn't have a loader on it. I'd comment it out and test whether or not it is actually necessary